setup.py

```diff
@@ -41,10 +41,6 @@
         "Intended Audience :: System Administrators",
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 3.4",
-        "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
```

Comment:
```suggestion
```

also remove 3.6 as we are only supporting python versions still in support. 

(3.7 just went out of support, but there isn't anything that really needs to move off of it so we can continue supporting it until a new feature requires only newer versions.)